tests/test_functions_call.py

```
@@ -0,0 +1,184 @@
+        temp_path = f.name
+    
+    yield temp_path
+    
+    if os.path.exists(temp_path):
+        os.unlink(temp_path)
+
+
+@pytest.fixture
+def sample_poses():
+    """Generate a list of sample pose transformation matrices"""
+    from utils import compose, euler_ZYX_to_R
```

Ruff (0.14.10)

Redefinition of unused `compose` from line 14

Remove definition: `compose`

(F811)